axum/src/handler/mod.rs

```diff
@@ -224,6 +224,23 @@ macro_rules! impl_handler {
 
 all_the_tuples!(impl_handler);
 
+mod private {
+    // Marker type for `impl<T: IntoResponse> Handler for T`
+    #[allow(missing_debug_implementations)]
+    pub enum IntoResponseHandler {}
```

Comment:
I don't think this needs to have `pub` visibility? At least on my v0.6.x based branch, things compiled with a private marker type.

Reply:
It didn't actually work when I tried using it

```
error: type `handler::IntoResponseHandler` is private
  --> examples/hello-world/src/main.rs:12:40
   |
12 |     let app = Router::new().route("/", get(StatusCode::OK));
   |                                        ^^^ private type
```

Reply:
What, at the usage site?? 😱

Reply:
yeah that confused me as well. Haven't seen that before.